README.md

```diff
@@ -6,6 +6,15 @@ This repository contains the code for the front-end portal to the NICE (Network
 
 The [Network Insight Collaboration Environment](https://digitalsupplychainhub.uk/showcase/critical-minerals-flagship/) aims to demonstrate that an innovative approach, based on new and existing technology, will allow supply chains to share insights and data across multiple data platforms, enhancing the efficiency of the supply chain network. It forms part of Critical Minerals, one of the flagships of the Digital Supply Chain Hub run by Made Smarter Innovation.
 
+### Usage
+
+To run the full stack, use:
+```
+docker-compose up --build
```

Comment:
Add -d flag as well?

Reply:
Personally, I don't encourage using detached mode, as it hides all the logs. That being said, anyone can run in detached mode if they want to.